### PR TITLE
Single Typo

### PR DESCRIPTION
**Pull request by Procyonae**
#### Summary
None

#### Describe the solution
Removes an extra comma

#### Describe alternatives you've considered
Letting someone get purp name

### CODE REVIEW

**Review by NetSysFire — APPROVED** [review]
Tbh I would reword the entire sentence since it sounds weird but it is a valid fix regardless

**Comment by NetSysFire** [comment]
I mean something like this:

>Zombie animals aren't good for eating, but sometimes you might find usable fur on 'em.

File: data/json/npcs/refugee_center/surface_visitors/NPC_hunter.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
       "Spot your prey before something nastier spots you.",
       "I've heard that cougars sometimes leap.  Maybe it's just a myth.",
       "The Jabberwock is real, don't listen to what anybody else says.  If you see it, RUN.",
-      "Zombie animal meat isn't good for eating, but sometimes you, might find usable fur on 'em.",
+      "Zombie animal meat isn't good for eating, but sometimes you might find usable fur on 'em.",
```
Comment (Procyonae) [inline data/json/npcs/refugee_center/surface_visitors/NPC_hunter.json:70]:
```suggestion
      "Zombified animals aren't good for eating, but sometimes you might find usable fur on 'em.",
```
